jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -425,14 +448,26 @@ frontend https-in
   <%- if properties.ha_proxy.frontend_config -%>
     <%= p("ha_proxy.frontend_config") %>
   <%- end -%>
-  <%- if_p("ha_proxy.cidr_whitelist") do -%>
```

Comment:
Allowlisting and blocklisting are mutually exclusive, right?
Moving the allowlist part down could theoretically change behaviour when both are active...